<?xml version="1.0" encoding="UTF-8"?>

<launch>
  <arg name="follower_model_name" default="turtle2" />
  <arg name="model_to_be_followed_name" default="coke_can" />

  <node name="tf_listener_node" pkg="tf_package" type="tf_listener.py" args="$(arg follower_model_name) $(arg model_to_be_followed_name)" output="screen" />
</launch>

